seedlet-blocks/assets/theme.css

```diff
@@ -148,7 +148,7 @@
 /**
  * Links in Content
  */
-.is-root-container a:not(.wp-block-button__link):not([download]),
+.is-root-container a:not(.wp-block-button__link):not([download]):not([aria-label="Site title text"]),
```

Comment:
Note:

Targeting an aria-label attribute seems no bueno, but I couldn't think of a better way to handle the specificity conflict. I didn't see any other way of identifying the site title `a` tag.